src/Routers/AnalyticsRouter.js

```diff
@@ -1,17 +1,15 @@
 // AnalyticsRouter.js
 import PromiseRouter from '../PromiseRouter';
 
-// Returns a promise that resolves to an empty object response
-function ignoreAndSucceed(req) {
-  return Promise.resolve({
-    response: {}
-  });
+function handlePost(req) {
+  const analyticsController = req.config.analyticsController;
```

Comment:
Usually we try not to forward express request directly to the controllers, and be a bit more explicit